Tests/AppTests/GithubTests.swift

```diff
@@ -15,10 +15,12 @@
 @testable import App
 
 import Vapor
+import SwiftSoup
+import S3Store
 import XCTest
 
 
-class GithubTests: AppTestCase {
+class GithubTests: SnapshotTestCase {
```

Comment:
```suggestion
class GithubTests: AppTestCase {
```

Class doesn't seem to have gained any snapshot tests?